paydb_endpoint.py

```diff
@@ -342,6 +402,7 @@ def user_transactions():
     return jsonify(dict(txs=txs))
 
 @paydb.route('/transaction_create', methods=['POST'])
+@limiter.exempt
```

Comment:
why are you exempting these endpoints?

Reply:
how many transactions allowed per IP. Need to make sure that the customer(s) not being NAT'd. ie. using wifi in a coffee shop or you'd error

Reply:
removed the exmepts